package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-instagram-embed",
-  "version": "1.5.0",
+  "version": "2.0.0",
```

Comment:
please return this modification back to the original bacause I have something to check before release

Reply:
done